exampleSite/content/index/footer.md

```diff
@@ -0,0 +1,21 @@
++++
+fragment = "footer"
+#disabled = true
+date = "2016-09-07"
+lastmod = "2017-09-07"
+weight = 1200
+#background = ""
+
+linkTitle = "Link Title"
+logoTitle = "Logo Title"
+logo = "logo.svg"
+logoText = "Logo Subtext"
+descriptionTitle = "Description Title"
+description ="""
```

Comment:
Probably can be done as .Content too or?

Reply:
Yes definately